src/schemas/transfer-benefits/schema.js

```diff
@@ -0,0 +1,105 @@
+import definitions from '../../common/definitions';
+import _ from 'lodash';
+
+export default {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'APPLICATION FOR FAMILY MEMBER TO USE TRANSFERRED BENEFITS',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions,
+    'privacyAgreementAccepted',
+    'ssn',
+    'gender',
+    'date',
+    'fullName',
+    'address',
+    'phone',
+    'bankAccount',
+    'educationType',
+    'school',
+    'postHighSchoolTrainings',
+    'dateRange',
+    'nonMilitaryJobs'
+  ),
+  properties: {
+    privacyAgreementAccepted: {
+      $ref: '#/definitions/privacyAgreementAccepted'
```

Comment:
i think we should keep definitions, the address definition is used in a lot of places (school.address, veteranAddress, spouseAddress, and relativeAddress) plus some definitions use other definitions